apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget.js

```diff
@@ -21,35 +22,36 @@
     return (new LockScreenClockWidgetSetup(this));
   };
 
+  LockScreenClockWidget.prototype.updateFormatters = function() {
```

Comment:
Since you only set them once, and the Setup state would only execute once, you can rename it with `setupTimeFormatters`.